var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -18,6 +18,7 @@ class SuperluDist(CMakePackage):
 
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
This affects other packages [wrt dependencies] so I'd rather not do this now [with xsdk changes]